src/LanguageProcessorEngine/Configuration/BaseLanguageProcessorConfiguration.php

```diff
@@ -7,11 +7,30 @@
 
 class BaseLanguageProcessorConfiguration extends BaseConfiguration implements LanguageProcessorConfigurationInterface
 {
+    const TYPE_KEY = 'type';
```

Comment:
Let's just call this TYPE for simplicity.

```suggestion
    const TYPE = 'type';
```